framework-plugins/lisk-framework-chain-connector-plugin/package.json

```diff
@@ -31,6 +31,7 @@
 		"build:check": "node -e \"require('./dist-node')\""
 	},
 	"dependencies": {
+		"@liskhq/lisk-chain": "0.4.0-alpha.0",
```

Comment:
you can import from `chain` from `lisk-sdk` where you can find `Block` and `BlockHeader`